charts/kube-master/templates/reject-proxy-config.yaml

```diff
@@ -0,0 +1,42 @@
+{{ if .Values.controllerManager.rejectProxy.enabled }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "master.fullname" . }}-reject-proxy
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ include "master.fullname" . }}-reject-proxy
+rules:
+- apiGroups: [""]
+  resources: ["configmaps"]
+  verbs: ["get", "watch", "list"]
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ include "master.fullname" . }}-reject-proxy
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ include "master.fullname" . }}-reject-proxy
+subjects:
+- kind: ServiceAccount
+  name: {{ include "master.fullname" . }}-reject-proxy
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "master.fullname" . }}-reject-proxy
+  namespace: {{ .Release.Namespace }}
+data:
+# Configuration example
+#  config: |-
+#    blockedUrls:
```

Comment:
How about changing the config structure so that every block list entry is a separate ConfigMap entry. This makes dynamically updating the block list a lot simpler because individual entries can be targeted for addition/deletion.
e.g.:
```
data:
  neutron-lbaas: |
    method: POST
    url: https://someurl:443/v2.0/lbaas.*
    code:  int #should be optional defaulting to 403
    message: string #should be optional default to something generic
```

Reply:
I changed config format.